JournalCoreData/Model Controllers/EntryController.swift

```diff
@@ -9,61 +9,59 @@
 import Foundation
 import CoreData
 
-let baseURL = URL(string: "https://journal-syncing.firebaseio.com/")!
+let baseURL = URL(string: "https://journal-790a5.firebaseio.com/")!
 
 class EntryController {
     
+    
+    init() {
+        fetchEntriesFromServer()
+    }
+    
+    
     func createEntry(with title: String, bodyText: String, mood: String) {
-        
         let entry = Entry(title: title, bodyText: bodyText, mood: mood)
-        
-        put(entry: entry)
-        
         saveToPersistentStore()
+        put(entry: entry)
     }
     
     func update(entry: Entry, title: String, bodyText: String, mood: String) {
-        
         entry.title = title
         entry.bodyText = bodyText
         entry.timestamp = Date()
         entry.mood = mood
-        
-        put(entry: entry)
-        
         saveToPersistentStore()
+        put(entry: entry)
     }
     
     func delete(entry: Entry) {
-        
         CoreDataStack.shared.mainContext.delete(entry)
         deleteEntryFromServer(entry: entry)
         saveToPersistentStore()
     }
     
-    private func put(entry: Entry, completion: @escaping ((Error?) -> Void) = { _ in }) {
```

Comment:
we don't actually want to remove this error handling here. it's important to be able to have this available down the line.